include/constraints/RebarBondSlipConstraint.h

```diff
@@ -0,0 +1,160 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+// MOOSE includes
+#include "Coupleable.h"
+#include "EqualValueEmbeddedConstraint.h"
+#include "Moose.h"
+
+/**
+ * A RebarBondSlipConstraint enforces the constraint between concrete and
+ * reinforcing bars establishing a slip vs. bondstress relationship
```

Comment:
The term `bondstress` shows up a lot of places here. I think it should have a space or hyphen between the words, or maybe an underscore when it's a code keyword.